qdrant-landing/content/blog/qdrant-1.16.x.md

```diff
@@ -0,0 +1,294 @@
+---
+title: "Qdrant 1.16 - Tiered Multitenancy & Disk-Efficient Vector Search"
+draft: false
+slug: qdrant-1.16.x
+short_description: "v1.16 of Qdrant focuses on tiered multitenancy with tenant promotion and disk-efficient vector search."
+description: "v1.16 of Qdrant focuses on tiered multitenancy with tenant promotion, disk-efficient vector search with inline storage, and improved filtered vector search with ACORN."
+date: 2025-11-19T00:00:00-08:00
+author: Abdon Pijpelink
+featured: true
+tags:
+  - vector search
+  - disk-based vector search
+  - tiered multitenancy
+---
+
+[**Qdrant 1.16.0 is out!**](https://github.com/qdrant/qdrant/releases/tag/v1.16.0) Let’s look at the main features for this version:
+
+**Tiered Multitenancy:** An improved approach to multitenancy that enables you to combine small and large tenants in a single collection, with the ability to promote growing tenants to dedicated shards.
+
+**ACORN**: A new search algorithm that improves the quality of filtered vector search in cases of high filtering selectivity.
+
+**Inline Storage**: A new HNSW index storage mode that stores vector data directly inside HNSW nodes, enabling efficient disk-based vector search.
+
+Additionally, version 1.16 introduces a new conditional update API, facilitating easier migration of embedding models to a newer version. And, this version improved Qdrant's full-text search capabilities with a new `text_any` condition and ASCII folding support.
+
+## Tiered Multitenancy Using Tenant Promotion
+  
+![Section 1](/blog/qdrant-1.16.x/section-1.png)
+
+Multitenancy is a common requirement for SaaS applications, where multiple customers (tenants) share the same database instance. In Qdrant, when an instance is shared between multiple users, you may need to partition vectors by user. This is done so that each user can only access their own vectors and can’t see the vectors of other users. To implement multitenancy in Qdrant, there are two main approaches:
+
+- [Payload-based multitenancy](/documentation/guides/multiple-partitions/), which works well when you have a large number of small tenants. This causes practically no overhead. Quite the opposite: a query with a tenant payload filter can be faster than a full search.
+- [Shard-based multitenancy](/documentation/guides/distributed_deployment/#user-defined-sharding), designed for when you have a smaller number of larger tenants. This works well when each tenant requires isolation and dedicated resources. Separating tenants by shard prevents a classic noisy neighbor problem where a single high-volume tenant can force the cluster to scale for everyone, increasing costs and reducing performance for smaller tenants.
```

Comment:
Note that Shard-based multitenancy have a significant overhead for small tenants